nx-cloud/templates/cloud.yml

```diff
@@ -650,24 +631,16 @@ spec:
         - name: data
           persistentVolumeClaim:
             claimName: cloud-volume
-      securityContext:
-        runAsUser: 10000
-        runAsGroup: 10000
-        fsGroup: 10000
-        fsGroupChangePolicy: "OnRootMismatch"
+      {{- if .Values.fileServer.securityContext.enabled }}
```

Comment:
Some OpenShift environments are not too happy about specific user IDs or groups. Thus, exposing configuration to disable this securityContext.